paragon/_overrides.scss

```diff
@@ -17,14 +17,19 @@
 // Badges
 
 .badge {
-  @badge-pill;
+  font-family: $font-family-monospace;
+  text-transform: uppercase;
 }
 
 
 // Buttons
 
+.btn {
+  text-decoration: none;
+}
+
 .btn-link {
-  text-decoration: $link-decoration;
+  text-decoration: none;
```

Comment:
Is it intentional for .btn-link to not be underlined? In the past, we've been asked to underline .btn-link elements by UX and/or a11y since it doesn't have it by default. But this request hasn't always been consistent.

I'm game for either case, just that we're consistent about it :)

Reply:
This is intentional. Non-underlined links buttons are usually sitting next to a regularly styled one. My understanding is that in the context of a "nav" or place where it should be obvious that they're interactive the underline is unneeded

Reply:
👍 Got it. Yeah, in the past, we've been asked to underline link buttons that sit next to a regularly styled one.